pkg/pillar/scripts/device-steps.sh

```diff
@@ -133,8 +133,8 @@ touch "$WATCHDOG_PID/zedbox.pid" "$WATCHDOG_FILE/zedbox.touch"
 
 if [ -c $TPM_DEVICE_PATH ] && ! [ -f $CONFIGDIR/disable-tpm ]; then
 #It is a device with TPM, enable disk encryption
-    if ! $BINDIR/vaultmgr setupVaults; then
-        echo "$(date -Ins -u) device-steps: vaultmgr setupVaults failed"
+    if ! $BINDIR/vaultmgr setupDeprecatedVaults; then
```

Comment:
Where do we setup up the non-deprecated vault?

Reply:
I guess you are referring to /persist/vault handling. That is done when vaultmgr starts with "runAsService". The idea is that, at some point (when we no longer have /persist/config and /persist/img as vaults on any of the EVE devices), we can remove this block of code from device-steps.sh, and just retain the part where vaultmgr is started with runAsService (at that point, we no longer need that argument, we can simply start vaultmgr along with other agents).